package.json

```diff
@@ -115,13 +115,13 @@
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-react": "^7.21.1",
     "eslint-plugin-react-hooks": "4.6.0",
-    "husky": "8.0.2",
+    "husky": "8.0.3",
     "nyc": "15.1.0",
-    "prettier": "2.7.1",
+    "prettier": "2.8.1",
     "pretty-quick": "3.1.3",
     "standard-version": "9.5.0",
     "uuid": "9.0.0",
-    "wait-on": "6.0.1"
+    "wait-on": "7.0.1"
```

Comment:
Can this major update break our stuff?

Reply:
From the [release notes for v7.0.0](https://github.com/jeffbski/wait-on/releases/tag/v7.0.0) they updates dependencies which means support Node 12.x was dropped. This should not break our stuff.